posthog/hogql_queries/test/test_query_runner.py

```diff
@@ -47,10 +47,13 @@ def calculate(self):
             def _refresh_frequency(self) -> timedelta:
                 return timedelta(minutes=4)
 
-            def _is_stale(self, cached_result_package, lazy: bool = False, *args, **kwargs) -> bool:
+            def _is_stale(self, last_refresh: Optional[datetime], lazy: bool = False, *args, **kwargs) -> bool:
+                if not last_refresh:
+                    return False
```

Comment:
This is carried over here to the tests... In the original it's there because we have last_refresh as optional in the object that is passed around, hence typing-wise it might make it as `None` into the function.